client/landing/stepper/hooks/use-email-verified-param.ts

```diff
@@ -0,0 +1,5 @@
+import { useQuery } from './use-query';
+
+export function useEmailVerifiedParam(): string | null {
+	return useQuery().get( 'emailVerified' );
+}
```

Comment:
I think we've started to adopt a pattern of creating these small hooks. They're nice to look at, and no strong feelings on this, but I do wonder how much advantage there is to a custom hook like this vs just doing the following in the file where it's needed (ie, for this PR, in launchpad/index.tsx above): 

```
import { useQuery } from './use-query';

...
const emailVerified = useQuery().get( 'emailVerified' );
```

Reply:
Good point. I was getting used to creating these small hooks due to the patterns we see in the calypso codebase, but it seems much cleaner to add this one-liner instead of creating a new file.